### ISSUE TITLE
USER ISSUE: Cheat movement detected, but no cheating. 

### ISSUE THREAD

**Issue by saskiavanryt** [StrangeLoopGames/EcoIssues#21609]
\
**User Id: (Not logged user)**
**Version:** 0.9.3.2 beta release-217

**Steps to Reproduce:**
Logged on to server, attempted to pick up limestone that I was standing on. Error occurred. 

**Expected behavior:**
Let me pick up the limestone.

**Actual behavior:**
The following error message popped up:
```
Cheat

<size=60.00%>Cheat movement detected. Report the issue if you're not cheating.</size>
```

**Do you have mods installed? Does the issue happen when no mods are installed?:**
Big shovel mod installed. 

**Please provide log files. Instruction is here https://github.com/StrangeLoopGames/EcoIssues/wiki/Getting-the-Eco-game-log-file

**Comment by Martino80** [comment]
in version 9.3.3 it seems to be fixed

**Comment by WeaselDog** [comment]
Thanks for the report. This is fixed in 9.3.3.
QA team can we check it is fixed in 9.4 also?
